src/pages/home/library/Discography.tsx

```diff
@@ -0,0 +1,90 @@
+import { Box } from "@mui/material";
```

Comment:
The content in here should be similar to what was in `Library.tsx`, needed to extract this to handle routing in new `Library.tsx` file